config/pipelines/pacbio.yml

```diff
@@ -304,7 +304,7 @@ default: &default
     # The number suffix '1' is used to identify the schema version of message (created by the
     # message builder class defined in the message_class key) 
     # The schema version number should be present in bunny.yml for the same 'key'
-      avro_schema_version_1 : 
+      avro_schema_version_1: &avro_schema_version_1
```

Comment:
Do we need to keep this version as  only one version will picked up from bunny.yml  ?

Reply:
I'm not entirely sure. The tests are highly coupled with this key. For example, if I remove `avro_schema_version_1`, I would have to change all the tests as well. 

My understanding of this `yml` was that we would keep the message building configs corresponding to each schema in this file. Not sure if I'm correct or incorrect in that regard.

Reply:
Ok, it's not a big issue for now, but something to think about in future. 
I think the dependency with the tests is because we are keeping a stub response at spec/fixtures/volume_tracking_avro_response.json which is for version 1.

Reply:
Yeah that's correct. But because we have the schemas in RedPanda, and we are _not_ removing them, if we decide to revert back to an older schema, that mean we would have to change this `pacbio.yml` file if we remove the older keys (e.g., `avro_schema_version_1`). 

I always thought this part of the `pacbio.yml` file as a versioned history of the mapping between the schema and the message builder.